Upstream-Status: Inappropriate [embedded specific]

Mips needs -fPIC to compile properly. These tests are broken anyway as they test 
the build system architecture, not the target. This is a hack.

RP 10/6/2010

Index: src/config/Linux_All.mk
===================================================================
--- src.orig/config/Linux_All.mk	2010-06-10 22:09:43.000000000 +0100
+++ src/config/Linux_All.mk	2010-06-10 22:09:55.000000000 +0100
@@ -90,14 +90,10 @@
 # Use the editline library to provide line-editing support.
 JS_EDITLINE = 1
 
-ifeq ($(CPU_ARCH),x86_64)
 # Use VA_COPY() standard macro on x86-64
 # FIXME: better use it everywhere
 OS_CFLAGS += -DHAVE_VA_COPY -DVA_COPY=va_copy
-endif
 
-ifeq ($(CPU_ARCH),x86_64)
 # We need PIC code for shared libraries
 # FIXME: better patch rules.mk & fdlibm/Makefile*
 OS_CFLAGS += -DPIC -fPIC
-endif
